addons/weaponCfgFixes/CfgWeapons.hpp

```diff
@@ -0,0 +1,14354 @@
+class Mode_FullAuto;
+class Mode_Burst;
+class Mode_SemiAuto;
+class CfgWeapons {
+    class Default;
+    class RifleCore: Default {};
+    class Rifle: RifleCore {};
+    class Rifle_Base_F: Rifle {};
+    class Rifle_Long_Base_F: Rifle_Base_F {};
+    class PistolCore: Default {};
+    class Pistol: PistolCore {};
+    class Pistol_Base_F: Pistol {};
+    class Rifle_Short_Base_F: Rifle_Base_F {};
```

Comment:
Do these need full redefining, or can we do:
```SQF
    class Rifle_Base_F;
    class Rifle_Long_Base_F;
    class Pistol_Base_F;
    class Rifle_Short_Base_F;
```

I want to fuck with as few base classes as possilbe